src/_utils/ripple.js

```diff
@@ -3,7 +3,7 @@ export default function ripple(node, options = {}) {
   let props = {
     event: options.event || 'click',
     transition: options.transition || 150,
-    zIndex: options.zIndex || '9999',
+    zIndex: options.zIndex || '100',
```

Comment:
Why is it a string and not a number?

Reply:
uh, just because it was written that way, no particular reason